.pipelines/singletenancy/aks-swift/e2e-job-template.yaml

```diff
@@ -41,6 +41,8 @@ stages:
       CURRENT_VERSION: $[ stagedependencies.containerize.check_tag.outputs['CurrentTagManifests.currentTagManifests'] ]
       commitID: $[ stagedependencies.setup.env.outputs['EnvironmentalVariables.commitID'] ]
     condition: and(succeeded(), eq(variables.TAG, variables.CURRENT_VERSION))
+    pool:
```

Comment:
So are these getting changed for stages even if the job has a default specified ?

Reply:
If the job level specifies a pool then the stage level gets overwritten.